Formula/pdftk-java.rb

```diff
@@ -0,0 +1,24 @@
+class PdftkJava < Formula
+  desc "Port of pdftk in java"
+  homepage "https://gitlab.com/pdftk-java/pdftk"
+  url "https://gitlab.com/pdftk-java/pdftk/-/archive/v3.0.6/pdftk-v3.0.6.tar.gz"
+  sha256 "bcc19573cd2cc316f8bd2039f01dd1b3b717405a25c58db589bde198bc63068b"
+  head "https://gitlab.com/pdftk-java/pdftk.git"
+
+  depends_on "gradle" => :build
+  depends_on :java => "1.7+"
+
+  def install
+    system "gradle", "wrapper", "--gradle-version=5.0", "--no-daemon"
```

Comment:
Why does it fetch a wrapper instead of running on the normal gradle? I thought that by convention the wrapper is committed.

Reply:
Also, why specifically `5.0` instead of the latest version? (I'm a Gradle employee)

Reply:
I don't see any gradle artifact committed in the upstream repo.
I create a gradle wrapper since the [documentation says](https://gitlab.com/pdftk-java/pdftk#building-and-running-with-gradle) it's a tested version and I thought it would make builds more predictable.
Also, for some reason gradle 5.1+ stores the `.jar` under a different name:

- gradle 5.0 stores it under `build/libs/pdftk-all.jar`
- gradle 5.1+ stores it under `build/libs/pdftk.jar`

I can either then remove the gradle wrapper creation and always use the latest version of gradle, or use gradle wrapper and pin it to version 5.0.
How do you suggest to proceed?

Reply:
Always use the latest version of gradle. If upstream prefered another way they would have included a wrapper.